spec/spec_helper.rb

```diff
@@ -38,37 +38,34 @@
 
 GoodData::Environment.load
 
-base = Pathname(__FILE__).dirname.expand_path
-Dir.glob(base + 'helpers/*_helper.rb').each do |file|
-  require file
-end
-
-include GoodData::Helpers
-
 RSpec::Expectations.configuration.warn_about_potential_false_positives = false
 
 RSpec.configure do |config|
   config.deprecation_stream = File.open('deprecations.txt', 'w')
 
-  config.include BlueprintHelper
-  config.include CliHelper
-  config.include ConnectionHelper
-  config.include CryptoHelper
-  config.include CsvHelper
-  config.include ProcessHelper
-  config.include ProjectHelper
-  config.include ScheduleHelper
-  # config.include SchemaHelper
-
   config.filter_run_excluding :broken => true
 
   config.fail_fast = false
 
-  # if ENV['VCR_ON'].nil? || ENV['VCR_ON'].downcase == 'true' # VCR is enabled by default - set VCR_ON=false to disable
-  # TODO; TMA-1071: remove this when VCR is fixed
-  if false
+  if GoodData::Environment::VCR_ON
     require 'vcr_configurer'
-    skip_sleep = VcrConfigurer.vcr_record_mode == :none
+    skip_sleep = GoodData::Helpers::VcrConfigurer.vcr_record_mode == :none
+
+    base = Pathname(__FILE__).dirname.expand_path
+    Dir.glob(base + 'helpers/*_helper.rb').each do |file|
+      require file
+    end
+
+    include GoodData::Helpers
```

Comment:
warning: Style/MixinUsage: `include` is used at the top level. Use inside `class` or `module`. (rubocop)